src/dappsExplorer/NoResults.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import Touchable from 'src/components/Touchable'
+import { DappFilter } from 'src/dapps/types'
+import { currentLanguageSelector } from 'src/i18n/selectors'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  filter: DappFilter | null
+  removeFilter: () => void
+  testID: string
+}
+
+export function NoResults({ filter, removeFilter, testID }: Props) {
+  const { t } = useTranslation()
+  const language = useSelector(currentLanguageSelector)
+
+  const onPress = () => {
+    removeFilter()
+  }
+
+  return (
+    <View testID={`${testID}/NoResults`} style={styles.favoritesContainer}>
+      <Text style={styles.filterAppliedText}>
+        <Trans
+          i18nKey={'dappsScreen.emptyResults.message'}
+          tOptions={{ filter: filter?.name.toLocaleUpperCase(language ?? 'en-US') }}
+        >
+          <Text style={styles.filterText} />
+        </Trans>
+      </Text>
+      <View style={styles.removeFilterTouchableContainer}>
```

Comment:
should we add a margin vertical here? the filter is a little tight once the content spills to two lines

![Simulator Screen Shot - iPhone 14 Pro - 2023-03-08 at 12 50 23](https://user-images.githubusercontent.com/20150449/223706754-48c2c9f4-055f-43e4-8022-39fe1214b51e.png)

Reply:
Applying the margin vertical here is not working well when text is in one line. Hopefully after the text style refactor this will be less of an issue.